charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -127,9 +127,9 @@ spec:
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 8 }}
       {{- end }}
-      {{- if .Values.tolerations }}
+      {{- with include "common.tolerations" . }}
       tolerations:
-{{ toYaml .Values.tolerations | indent 8 }}
+        {{- . | nindent 8 -}}
```

Comment:
```sugestion
         {{- . | nindent 8 }}
```
If you render `test: test` toleration with a right line cleaner it could render like this:
```yaml
tolerations:
  test:testaffinity:
```
It is not going to happen because of lines 133 and 134 but be really careful when to and not to put line chompers.

We usually, while rendering, put it only on the left.

Reply:
Ditto.